unit_tests/test_views.py

```diff
@@ -34,9 +34,10 @@ def test_about(self):
         self.assertContains(response, '<p>footnote text') #make sure that footnote was rendered
 
     def test_links(self):
+        models.Static.objects.create(title='Links', text='### Links')
         response = self.client.get(reverse('links'))
         self.assertEqual(response.status_code, 200)
-        self.assertContains(response, '<h2>Links</h2>')
+        self.assertContains(response, 'Links')
```

Comment:
Last comment before I deploy to the test server: if you leave the html header tags in the assert here, you can verify that the markdown is being rendered to HTML. So you could check for "<h3>Links</h3>".